ui/k-info-panel/switch_info.kytos

```diff
@@ -39,6 +39,16 @@
             </tbody>
           </table>
         </div>
+      </k-accordion-item>            
+      <k-accordion-item title="Metadata actions"> 
+         <k-textarea title="Add metadata" icon="arrow-right" placeholder='Eg. {"bandwidth": 100, "port_name": "some_name"}' :value.sync="to_add"></k-textarea>
+         <div class="metadata_container">
+              <k-button title="Add metadata" :on_click="bt_add_metadata"></k-button>
+         </div>
+         <k-input title="Delete metadata" icon="arrow-right" placeholder="Eg. port_name" :value.sync="to_delete"></k-input>
```

Comment:
Same here, could you update the placeholder with `node_name` instead of `port_name`?

Reply:
Updated on commit [c032f056419466ae6673f04b7d2c08339dc81a4e](https://github.com/kytos-ng/topology/pull/74/commits/c032f056419466ae6673f04b7d2c08339dc81a4e)